service/controller/v1/prometheus/scrapeconfig.go

```diff
@@ -30,9 +30,11 @@ const (
 
 	// APIServerJobType is the job type for scraping Kubernetes API servers.
 	APIServerJobType = "apiserver"
+	// AWSNodeJobType is the job type for scraping aws-node Pods.
+	AWSNodeJobType = "aws-node"
 	// CadvisorJobType is the job type for scraping Cadvisor.
 	CadvisorJobType = "cadvisor"
-	// CalicoNodeJobType is the job type for scraping calico-node PODs.
+	// CalicoNodeJobType is the job type for scraping calico-node Pods.
```

Comment:
```suggestion
	// CalicoNodeJobType is the job type for scraping calico-node pods.
```

Nitpick, but if it's not an acronym, then lowercase I guess? That's how it's in k8s code.